src/main/java/uk/co/onsdigital/discovery/model/Variable.java

```diff
@@ -1,17 +1,6 @@
 package uk.co.onsdigital.discovery.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.JoinTable;
-import javax.persistence.ManyToMany;
-import javax.persistence.ManyToOne;
-import javax.persistence.NamedQuery;
-import javax.persistence.OneToMany;
-import javax.persistence.SequenceGenerator;
+import javax.persistence.*;
```

Comment:
I always thought my IDE settings expanded them. Turns out it collapses them with > 5 imports from a package. I have updated my IDE settings to not collapse them and updated the existing model classes.